lib/index.js

```diff
@@ -396,7 +396,7 @@ const Popover = createClass({
       !this.containerEl.contains(event.target) &&
       !this.targetEl.contains(event.target)
     )
-    if (isOuterAction) this.props.onOuterAction()
+    if (isOuterAction) this.props.onOuterAction(event)
```

Comment:
I was hiding the pop up on click outside the modal. I noticed that the pop up was closing to easily on tap on mobile. I was not able to scroll up or down if necessary.
So I wanted to close only when the mouse was used. This change let me use:

    _handleOuterAction = (e) => {
      if (e.type == 'mousedown')
        this._handleOnClose();
    }   

Why not put it in? It is backward compatible.